sym/13e747a6-3ce1-4722-a18e-4fc469c491b0/symbol.lp

```diff
@@ -0,0 +1,120 @@
+(librepcb_symbol 13e747a6-3ce1-4722-a18e-4fc469c491b0
+ (name "ADS8868")
+ (description "16-Bit 500kSPS 8-Channel Single-Supply SAR ADC With Bipolar Input Ranges")
+ (keywords "spi,adc,bipolar")
+ (author "Matej K.")
+ (version "0.1")
+ (created 2020-04-11T21:57:38Z)
+ (deprecated false)
+ (category a621d579-8715-4edf-816e-5715f97cd2d7)
+ (pin 809f7f16-6b76-4d9d-94bd-bfd3b673c67e (name "AIN0_P")
```

Comment:
More of a comment, not a change request: The datasheet inconsistently names the analog pins (e.g. `AIN_6P` but `AIN2_GND`). I guess `AINx_(P|GND)` makes most sense.

Edit: Woah, that datasheet is horrible. Sometimes they even mix up the pin names in the descriptions...